soh/soh/Enhancements/randomizer/3drando/hints.cpp

```diff
@@ -779,6 +800,8 @@ void CreateAllHints() {
 
   CreateGanonText();
   CreateAltarText();
+  CreateDampesDiaryText();
+  CreateWarpSongTexts();
```

Comment:
Something to keep in mind here, this `CreateAllHints()` only gets called if gossip stone hints are turned on. I recently moved `CreateWarpSongTexts()` outside of this func for that very reason.

https://github.com/HarbourMasters/Shipwright/blob/4d8c70fb1f9bd1cfba0b65fb3e38e2aac39a65a2/soh/soh/Enhancements/randomizer/3drando/fill.cpp#L1069-L1079

Also worth mentioning, I created #2217 which could probably be addressed here if we want.

Reply:
> Also worth mentioning, I created https://github.com/HarbourMasters/Shipwright/issues/2217 which could probably be addressed here if we want.

I'll handle it as well as making separating altar hints from gossip stone hint generation.